README.md

```diff
@@ -18,99 +18,116 @@ Other supported formats are listed below.
 
 ### Common parameters
 
-* `user id` - enter the SQL Server Authentication user id or the Windows Authentication user id in the DOMAIN\User format. On Windows, if user id is empty or missing Single-Sign-On is used. The user domain sensitive to the case which is defined in the connection string.
-* `password`
-* `database`
-* `connection timeout` - in seconds (default is 0 for no timeout), set to 0 for no timeout. Recommended to set to 0 and use context to manage query and connection timeouts.
-* `dial timeout` - in seconds (default is 15), set to 0 for no timeout
-* `encrypt`
-  * `disable` - Data send between client and server is not encrypted.
-  * `false` - Data sent between client and server is not encrypted beyond the login packet. (Default)
-  * `true` - Data sent between client and server is encrypted.
-* `app name` - The application name (default is go-mssqldb)
+- `user id` - enter the SQL Server Authentication user id or the Windows Authentication user id in the DOMAIN\User format. On Windows, if user id is empty or missing Single-Sign-On is used. The user domain sensitive to the case which is defined in the connection string.
```

Comment:
Please revert all changes to the README.md, except for actual text changes.

If you want to change the markdown format or if a tool is doing this for you, you can propose that in a new change.

Reply:
Done